upstream.go

```diff
@@ -1089,6 +1211,8 @@ func (uc *upstreamConn) handleMessage(msg *irc.Message) error {
 		// Ignore
 	case irc.RPL_MOTDSTART, irc.RPL_MOTD, irc.RPL_ENDOFMOTD:
 		// Ignore
+	case irc.RPL_LISTSTART:
```

Comment:
Do we need to send an initial `RPL_LISTSTART` to clients?

Reply:
Probably not, it's optional and marked as deprecated in RFC 2812